Products/Brass-Beta/gWASM.md

```diff
@@ -650,15 +653,14 @@ fn main() {
 }
 ```
 
-Here, `dispatcher` is part of our `gwasm-api` is essentially speaking, it
-ties all 3 stages together.
+The `dispatcher` module is a core of our `gwasm-dispatcher` crate and ties all 3 stages of the gWASM app.
```

Comment:
```suggestion
The `dispatcher` module is the core of our `gwasm-dispatcher` crate and ties together all 3 stages of the gWASM app.
```

Reply:
there is together in very next line, so I'd avoid it here